### DISCUSSION TITLE
Warning: Installing shim for bottle import redirects (using Python=3.12.2 and bottle=0.12.25)

### DISCUSSION THREAD

**Discussion by mnm000** [brentvollebregt/auto-py-to-exe#476] [Q&A]
Hello 
Im having issue running the auto-py-to-exe..
Keep getting Warning: Installing shim for bottle import redirects (using Python=3.12.2 and bottle=0.12.25) and not running

**Comment by brentvollebregt** [comment]
When you say "and not running", what exactly do you mean?

- Does the terminal do nothing
- Does the application exit
- Does the UI appear and not load
- Does something else happen?

**Reply by mnm000** [reply]
terminal do nothing,UI not appear  I reinstall auto-py-to-exe. many time still having issues

![image](https://github.com/brentvollebregt/auto-py-to-exe/assets/76624177/9ef61f5e-c5e5-4dd4-b4e3-9309ff06c0d7)

**Reply by brentvollebregt** [reply]
Nice, thanks for the screenshot. If you run `auto-py-to-exe --no-chrome`, what happens?
